rules/Proj0025.md

```diff
@@ -3,14 +3,16 @@ parent: General
 ancestor: MSBuild
 ---
 
-# Proj0025: Migrate from ruleset file to .editorconfig file
-Microsoft deprecated the use of `.ruleset` in favor of `.editorconfig` files.
+# Proj0025: Migrate from ruleset file to .globalconfig file
+Microsoft deprecated the use of `.ruleset` in favor of `.globalconfig` files.
 In Visual Studio by clicking the file, it will automatically pop up a dialog to
 convert the ruleset.
 
 Alternatively, you can download the converter yourself from [NuGet](https://www.nuget.org/packages/Microsoft.CodeAnalysis.RulesetToEditorconfigConverter).
```

Comment:
The rule is (now) about .globalconfig. But both links here refer to .editorconfig. I would order it differently or put some extra text explaining why.

Reply:
I thought the next sentence was about why the refer to `.editorconfig` but we advise `.globalconfig` anyway. But if you think that could/should even be specified in more detail, I hope you can come up with a suggestion on who to do that.